client/styles/styles.css

```diff
@@ -9,6 +9,74 @@ body {
   color: #2F195F;
 }
 
+.header {
```

Comment:
Should be better aligned with design, please check the desig in figma

- height should be 90px
- no top padding
- do we need % padding ?
- should be better aligned in the center 
- Sign in button too small
- Text in sign-in button 24px iso 18px

Reply:
thank you for the review